string-manipulation-1/src/is-upper-cased.js

```diff
@@ -0,0 +1,15 @@
+/* exported isUpperCased */
+function isUpperCased(word) {
+  for (var i = 0; i < word.length; i++) {
+    if (word[i].charCodeAt(0) >= 97) {
+      return false;
+    }
```

Comment:
Same thing here, you should make use of `toUpperCase` and `toLowerCase` as needed.